src/main/views/lip-or-representative.njk

```diff
@@ -12,6 +14,19 @@
 {% block content %}
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
+
+          {% if noRadioButtonSelectedError %}
```

Comment:
Similar to before. If possible please line up the start and the end of the if block